lib/main.js

```diff
@@ -37,6 +37,23 @@ function (moment, Router, L, GUI, numeral) {
           }
       }
 
+      var badNodes = []
```

Comment:
This is not the responsibility of the client!
Please remove it! If you have problems with this stuff, report it at hopglass/hopglass-server!